static/lib/composer.js

```diff
@@ -311,6 +311,16 @@ define('composer', [
 			$(this).attr('disabled', true);
 			post(post_uuid);
 		});
+		
+		postContainer.keypress(function (event) {                                 
+			var keyCode = (event.which ? event.which : event.keyCode);
+			if (keyCode === 10 || keyCode == 13 && event.ctrlKey) {
```

Comment:
I think this should be `if ((keyCode === 10 || keyCode == 13) && event.ctrlKey) {` as per https://stackoverflow.com/a/9343095/583363

Reply:
Indeed @barisusakli